ansible/ansible.cfg

```diff
@@ -26,6 +26,9 @@ transport      = smart
 module_lang    = C
 max_diff_size  = 512000
 
+# enable stdout plugin to debug
+stdout_callback = debug
+
```

Comment:
Please make this change in a separate PR. Thanks

Reply:
updated, thanks.